README.md

```diff
@@ -70,6 +70,23 @@ The benchmark can be found from [here](docs/en/03-benchmark/benchmark.md)
 | Adreno GPU        | -                                                               | -                                       | -        | ✔️ncnn<br>✔️SNPE |
 | Hexagon DSP       | -                                                               | -                                       | -        | ✔️SNPE           |
 
+[![Build Status][pass-backend-ort]][ci-backend-ort]
```

Comment:
May put the build results in the above matrix

Reply:
> May put the build results in the above matrix

Please take a look at the `README.md`. I tried putting the icons there, but it's kind of ugly. LOL

Reply:
Try to put the build badge at the beginning

Reply:
> Try to put the build badge at the beginning

Moved the badges to the front of each framework

Reply:
Besides `passing` and `failing`, is there another status like `TODO`?

Reply:
> Besides `passing` and `failing`, is there another status like `TODO`?

These build status icons provided by img.shileds.io need to be specified by an exact build action yml URL. Therefore, there are only four states including `passing`, `falling`, `no status`, and cannot find such an action. I tried to put a `no status` that links to rknpu2-gcc in front of those without action. Not ideal and may cause misleading to others.

Reply:
Let's try to remove "✔️"